src/resources/js/download.js

```diff
@@ -218,4 +246,16 @@ function enableFields() {
 	window.onbeforeunload = null;
 }
 
-var downloadButtonHtml; // to restore button to its original contents
\ No newline at end of file
+var downloadButtonHtml; // to restore button to its original contents
+
+function getAllPackageFromURLSearch() {
+	return window.location.search ? new URLSearchParams(window.location.search).getAll('package') : [];	
```

Comment:
```suggestion
new URL(window.location.href).searchParams.getAll("package")
```

And so you maybe not need a dedicated function

Reply:
Done, Thanks ! :)